.github/workflows/e2e-android-detox.yml

```diff
@@ -0,0 +1,80 @@
+name: Detox E2E Android Tests PR
+
+on:
+    pull_request:
+      branches:
+        - main
+      types:
+        - labeled
+
+concurrency:
+    group: "${{ github.workflow }}-${{ github.event.pull_request.number }}-${{ github.event.label.name }}"
+    cancel-in-progress: true
+
+jobs:
+    build-android-apk:
+        if: github.event.label.name == 'E2E Android tests for PR'
+        runs-on: ubuntu-latest-8-cores
+        env:
+            ORG_GRADLE_PROJECT_jvmargs: -Xmx8g
+        steps:
+            - name: Prune Docker to free up space
+              run: docker system prune -af
+
+            - name: Remove npm Temporary Files
+              run: |
+               rm -rf ~/.npm/_cacache
```

Comment:
Do we need these steps? We're not doing the same when building Android in other places.

Reply:
This is specific to having space when build command runs. it constantly runs out of memory in CI. Just added as precautionly step.